TabularEditor/TreeViewAdv/Tree/TreeColumn.cs

```diff
@@ -141,6 +141,17 @@ public bool IsVisible
 			}
 		}
 
+        private bool _hideOnFullRow = true;
+        [DefaultValue(true)]
+        public bool HideOnFullRow

```

Comment:
Shouldn't you be setting this property to False on the new severity column? I don't see that you're doing that anywhere.

Reply:
Ah never mind - for some reason, I didn't see the diff on BPAForm.Designer.cs.